handout/forms/getting-started.md

```diff
@@ -3,17 +3,19 @@
 
 ### Opt-In APIs
 Before we dive into any of the form features, we need to do a little bit of housekeeping.
-If you are using a RC5 or above of Angular 2, we need to bootstrap our application using the new `FormsModule` and/or `ReactiveForms` module.
+If you are using a RC5 or above version of Angular 2, we need to bootstrap our application using the new `FormsModule` and/or `ReactiveFormsModule`.
```

Comment:
Since 2.0 has been out for a while we should skip the RC5 bit and just explain this as the normal bootstrapping process